frontend/src/components/CoduoDocs/CoduoDocs.tsx

```diff
@@ -0,0 +1,180 @@
+import DocsImage from '@/components/CoduoDocs/DocsImage/DocsImage';
+import ContentBox from '@/components/CoduoDocs/FloatingSidebar/ContentBox';
+import FloatingSidebar from '@/components/CoduoDocs/FloatingSidebar/FloatingSIdebar';
+import Quote from '@/components/CoduoDocs/Quote/Quote';
+import SourceCode from '@/components/CoduoDocs/SourceCode/SourceCode';
+
+import useHashScroll from '@/hooks/Scroll/useHashScroll';
+
+import { ABOUT_PAIR_PROGRAMMING, DOCS_IMAGES, START_CONTENT } from '@/constants/coduoDocs';
+
+import * as S from './CoduoDocs.styles';
+
+const CoduoDocs = () => {
+  const { activeSection } = useHashScroll();
+
+  return (
+    <>
+      <FloatingSidebar>
+        <ContentBox activeSection={activeSection} title="시작하기" contents={START_CONTENT} />
+        <ContentBox activeSection={activeSection} title="페어 프로그래밍에 대해" contents={ABOUT_PAIR_PROGRAMMING} />
+      </FloatingSidebar>
+
+      <S.Layout>
+        <S.ParagraphContainer aria-labelledby={START_CONTENT[0].id}>
+          <S.Title id={START_CONTENT[0].id}>코딩해듀오 시작하기</S.Title>
+          <S.Content>
+            코딩해듀오 문서에 오신 것을 환영합니다! 코딩해듀오는 페어 프로그래밍을 처음 접하는 사용자가 페어
+            프로그래밍을 시작하기 위해 필요한 모든 것을 제공하는 서비스입니다. 여기에서는 코딩해듀오를 어떻게 시작할 수
+            있는지 소개합니다.
+          </S.Content>
+
+          <DocsImage
+            information="1. 방 생성하기"
+            src={DOCS_IMAGES.createRoom}
+            alt="create-room"
+            webpSrc={DOCS_IMAGES.createRoomWebp}
+          >
+            <Quote text="가장 먼저 방을 생성해주세요. 방 만들기 버튼을 누르면 페어 프로그래밍을 진행할 방이 생성됩니다" />
+          </DocsImage>
+
+          <S.Sentence>
+            <S.Content>코딩해듀오는 원활한 페어 프로그래밍 진행을 위해 연습 미션을 제공하고 있습니다.</S.Content>
+
+            <Quote href="#start-free" linkText="여기부터 읽기" text="미션 없이 자유롭게 시작하려면?" />
+          </S.Sentence>
+        </S.ParagraphContainer>
+
+        <S.ImageContainer aria-labelledby={START_CONTENT[1].id}>
+          <S.ParagraphContainer>
+            <S.Subtitle id={START_CONTENT[1].id}>미션과 함께 시작하기</S.Subtitle>
+            <Quote
+              href="https://github.com/coduo-missions"
+              linkText="미션 리포지토리로 이동"
+              text="미션 리포지토리에서 미션을 미리 확인해 보세요."
+              isNewBrowserOpen={true}
+            />
+          </S.ParagraphContainer>
+          <DocsImage
+            information="1. 미션과 함께 시작하기"
+            src={DOCS_IMAGES.startWithMission}
+            alt="start-with-mission"
+            webpSrc={DOCS_IMAGES.startWithMissionWebp}
+          />
+          <DocsImage
+            information="2. 미션 선택하기"
+            src={DOCS_IMAGES.selectMission}
+            alt="check-branch-created"
+            webpSrc={DOCS_IMAGES.selectMissionWebp}
+          />
+          <DocsImage
+            information="3. branch 생성하기"
```

Comment:
```suggestion
            information="3. 브랜치 생성하기"
```

여기만 단어가 영어로 되어 있는 게 이상해서 바꾸면 좋을 것 같아요 👀